internal/collected/github/enterprise.go

```diff
@@ -0,0 +1,43 @@
+package githubcollected
+
+import (
+	"github.com/Legit-Labs/legitify/internal/common/namespace"
+	"github.com/Legit-Labs/legitify/internal/common/permissions"
+)
+
+type Enterprise struct {
+	MembersCanChangeRepositoryVisibilitySetting string `json:"visibility_change_disabled"`
+	EnterpriseName                              string `json:"name"`
+	Url                                         string `json:"url"`
+	Id                                          int64  `json:"id"`
+	UserRole                                    string
+}
+
+func NewEnterprise(MembersCanChangeRepositoryVisibilitySetting string, Name string, Url string, Id int64, admin bool) Enterprise {
+	UserRole := ""
+	if admin {
+		UserRole = permissions.EnterpriseAdmin
```

Comment:
How does it work? the user can be enterprise owner meanwhile the token provided doesn't contain 'admin:enterprise' scope